client/dashboard/data/types.ts

```diff
@@ -78,13 +77,15 @@ export interface Site {
 	};
 	plan?: SitePlan;
 	active_modules?: string[];
+	capabilities: Record< string, Record< string, boolean > >;
```

Comment:
Shouldn't this be just `Record<string, boolean>`?

Or maybe for now something like 

```ts
interface SiteCapabilities {
  manage_options?: boolean;
}

...

capabilities: SiteCapabilities;
```